docs/conf.py

```diff
@@ -206,3 +210,25 @@
     # This page is under login
     r"https://readthedocs\.org/accounts/gold",
 ]
+
+
+# Config values for sphinxcontrib-mermaid are handled differently
+# If running on a local environment, we render "raw" and bundle mermaid-js from CDN to render diagrams in the browser.
+# This is to avoid requiring the @mermaid-js/mermaid-cli package
+#
+# This can run locally but the following issues need to be solved:
+# a) Read the Docs Docker images need at least libasound2 to be installed for puppeteer to work.
+# b) A <style> with FontAwesome needs to be injected into the SVG's DOM.
+if False and os.environ.get("READTHEDOCS"):
```

Comment:
```suggestion
if os.environ.get("READTHEDOCS") == "True":
```

Reply:
Thanks for the comments! It's hard to see in such a lengthy thread, but I did intend to remove all of this since we can't add the plugin with much benefit.

I might turn it into an issue where we can discuss the approach and see if there's anyone else out there who'd benefit.